src/assets/main.css

```diff
@@ -123,6 +131,52 @@
   .lux-sortable-drag {
     @apply drop-shadow-md;
   }
+
+  .lux-navbar {
+    @apply h-14 flex bg-white shadow-header z-10 shrink-0;
+  }
+
+  .lux-navbar-dropdown.lang .lux-dropdown-btn {
+    @apply w-16 h-full after:font-icons after:content-['\e012'] after:text-3xl border-none lux-outlined-none;
+    color: var(--color-navbar-default);
+  }
+
+  .lux-navbar-dropdown.lang .lux-dropdown-btn.expanded {
+    @apply after:content-['\e02c'];
+    color: var(--color-navbar-default);
+  }
+
+  .lux-navbar-dropdown.lang .lux-dropdown-btn > span {
+    @apply hidden;
+  }
+
+  .lux-navbar-dropdown.lang .lux-dropdown-btn:hover {
```

Comment:
Wondering if we shouldn't rather handle non-generic cases with tailwind's standard classes in the templates, to prevent this sheet from growing too much!?

Reply:
@tkohr I have left all styles in the main.css but I have made them a bit more generic: eg. `.lux-navbar-dropdown.lang` simply becomes  `.lux-navbar-dropdown`. This navbar dropdown style will be used for tools like Draw and Measure, this is why I kept them in the main.css. Is it better now?
I agree the whole styling needs a little rework.

Reply:
Thanks for the rework and making the `main.css` content more generic! Looks better to me (the diff is just harder to see without a separate commit:-)

Reply:
Next time i'll do a new commit instead of a commit amend. Sorry :)